Content: rewrite git prep content for week 1 (#168)
CodeYourFuture/curriculum: content/induction/blocks/github-poets/index.md

+++
title = '📖 Git and Github for poets'
headless = true
time = 15
facilitation = false
vocabulary=[""]
emoji= '🧩'
[objectives]
1='Double check Github account is setup'
2='Double check Course planner is setup'
+++

> This playlist will give you an introduction to Git & Github which are two tools we'll be using during the induction week.

Reviewer: ```suggestion
This playlist introduces Git & Github: tools we'll be using from now on.
```

> These videos cover concepts that you'll be learn slowly over this course as you need them. Be sure to use these videos in the future when you need them.

Reviewer: ```suggestion
These videos cover concepts that you'll learn slowly over this course. Bookmark these videos and come back in the future when you need them.
```

> For now, you only need to watch the first video:

Reviewer: ```suggestion
For now, watch only the first video:
```
